### PR TITLE
Update Postgres to 15.10

### PR DESCRIPTION
**Pull request by sir-sigurd**
- [x] changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ Optional release notice.
 
 ## [Unreleased] - YYYY-MM-DD
 
-Optional release notice.
+To update to this version, version 1.0.0 must be deployed first.
```
Comment (sir-sigurd) [inline CHANGELOG.md:13]: Updating postgres to 15.10 requires 15.x already deployed and 1.0.0 is the first *release* having it

Comment (sir-sigurd) [reply CHANGELOG.md:13]: maybe we should include such update compatibility for each release? 🤔

**Review by nl0 — APPROVED** [review]
lgtm

File: CHANGELOG.md (hunk 1 above)
Comment (nl0) [reply CHANGELOG.md:13]: you mean requiring users to upgrade one step at a time?

Comment (sir-sigurd) [reply CHANGELOG.md:13]: yes

**Pull request merged**